### ISSUE TITLE
Enhancement Request: Support for Sending Data to Multiple Mixpanel Projects

### ISSUE THREAD

**Issue by Vitaly-A-Ivanov** [asakura/mixpanel_api_ex#24]
I would like to request an enhancement to the `mixpanel_api_ex` library to support sending data to multiple Mixpanel projects based on specific criteria. Currently, the library is designed to interact with a single Mixpanel project using a configured token.

**Use Case:** 
In certain scenarios, it is necessary to send data to different Mixpanel projects based on the data itself, such as distinguishing between projects for different clients or segments.

**Requested Functionality:** 
This enhancement request aims to enable the library to send data to multiple Mixpanel projects simultaneously. Ideally, this functionality would allow developers to configure and manage multiple instances of the `mixpanel_api_ex` client, each with its own Mixpanel project token, and direct data to the appropriate instance based on specific criteria, such as the data content.

**Example:**
A potential use case is an application that serves multiple clients, and each client has its own Mixpanel project. Based on the client associated with a user’s action, the application should send the data to the corresponding Mixpanel project.

```
config :mixpanel_api_ex, :config,
   project_name: : project_one,
   project_token: "<Put Project Token here>",
   active: true
   
config :mixpanel_api_ex, :config,
   project_name: : project_two,
   project_token: "<Put Project Token here>",
   active: true
   
   
Mixpanel.track(: project_one, event, properties, opts)
Mixpanel.track(: project_two, event, properties, opts)
```

**Performance and resource implications:** 
Providing the ability to manage multiple Mixpanel clients should be done in a way that is efficient and does not introduce significant overhead.

Please consider this enhancement request to make the `mixpanel_api_ex` library even more versatile for handling diverse use cases involving multiple Mixpanel projects. 🚀 

Thank you for your attention to this request.

**Comment by asakura** [comment]
This feature has been already implemented actually. I'm going to release a new version soon.

**Comment by Vitaly-A-Ivanov** [comment]
Will try tomorrow and let you know asap 🔥🔥🔥 Many thanks!

**Comment by Vitaly-A-Ivanov** [comment]
@asakura Well, I was able to setup new version and run multiple instances. It works like a charm and sending events accordingly 👏 🥳 💯 🚀 

Only one issue I had when I should always explicitly define an adapter for each config, as it does not pick `Mixpanel.HTTP.HTTPC` by default  like it was before. If I am not providing the adapter for `config :mixpanel_api_ex`, then it compiles fine, but producing the error as soon an event being triggered

```
** (UndefinedFunctionError) function HTTP.HTTPC.get/3 is undefined (module HTTP.HTTPC is not available)
......
......
```

**Comment by asakura** [comment]
@Vitaly-A-Ivanov just let you know that it was fixes in 1.2.1

**Comment by Vitaly-A-Ivanov** [comment]
@asakura thanks for the update! Just to let you know that new your `VERSION` file has not released for some reason and it causing the bug:

```
** (File.Error) could not read file "VERSION": no such file or directory
    (elixir 1.15.5) lib/file.ex:358: File.read!/1
```

**Comment by asakura** [comment]
@Vitaly-A-Ivanov I hope that 1.2.2 fixes that :)

**Comment by Vitaly-A-Ivanov** [comment]
@asakura I can confirm all issues has been resolved in 1.2.2 version 🚀 Thanks so much!
